teams/developer-relations/guide-to-being-a-manager/README.md

```diff
@@ -90,49 +90,110 @@ With regards to this checklist, reviewers should do the following:
+
+To label medium cards, simply add another number associated with its order.   
+In the above example, **1-1.md is the first medium card associated with the first hard card**, and   
+**2-1.md is the first medium card associated with the second hard card**.
+
+To label easy cards, add another number associated with its order \(3 numbers total\).  
+In the above example, **1-1-1.md is the first easy card associated with the first medium card associated with the first hard card.** Can you translate 2-1-1.md into plain English?
+
+#### Card Length
+
+Cards should be concise. Ideally, you should not have to scroll more than once to view the whole card. Stuffing a ton of information in a single card can be too overwhelming, so try to keep things brief and split up topics as much as possible. 
+
+#### Writing Conventions
+
+Last but certainly not least, proper writing conventions such as grammar, punctuation, capitalization, etc. should be utilized. 
+
+### Stages
+
+Here's an in-depth breakdown of how cards progress in the BitProject system. 
 
 #### Stage 1 - Starting Content
 
-* [ ] Content of cards is more accessible to beginners
-* [ ] Micro to Macro Principle
-* [ ] Titles for Concepts associated with each Card
+**Card content should be accessible to beginners**, meaning someone with no prior knowledge should be able to follow the content. This goes hand-in-hand with using a  conversational writing style.
+
+Next up is what we call the **Micro to Macro Principle**. For labs and activities, please start at a micro level and then move up. For example, students should not code the main\(\) function first. As a rule of thumb, you should never have to utter the words “this function will be coded in a later part”. Every hard card should work on its own and not have to rely on later work for it to work out.
+
+![Micro &#x2014;&amp;gt; Macro!](../../../.gitbook/assets/smalltobig.jpeg)
+
+**Images**! Make liberal use of images \(like the one above\) in your cards. It's much easier to visualize a concept than to read three paragraphs of text. Obviously, text is necessary, but pairing text with visuals is immensely helpful. Also, images help break up blocks of text, making it easier on the eyes. Please use copyright-free images, Pexels is a great source. Make sure images are embedded using &lt;img&gt; tags and not URLs.
+
+Activities and Labs will feature coding tasks for students to complete. For these, developers should implement what we call **checkpoints**—Python console output that serves as a frame of reference for students to make sure their code is doing what it's supposed to be doing. 
+
+**Titles**. These should be included in HTML comments at the top of each card, not in the file name itself \(remember, cards are named numerically\). Along with titles, **concepts** covered in the card should also be contained in HTML comments. Here's an example:
```

Comment:
titles and concepts are actually now in READMEs, but this doesn't have to be mentioned in Stage 1 (it is a Stage 3 item), you can delete it